super_gui/src/components/DoorManUI.tsx

```diff
@@ -70,21 +115,19 @@ const DoorManUI = () => {
                   </div>
                   <div className="flex flex-col mb-3">
                     <label className="mb-1 font-body text-charcoal">
-                      Locker Number (1-7)
+                      Locker Number (1-5)
                     </label>
                     <select
                       className="border border-beige rounded p-2"
                       value={selectedLocker}
                       onChange={(e) => setSelectedLocker(e.target.value)}
                     >
                       <option value="">Select a locker</option>
-                      <option value="1">Locker 1</option>
-                      <option value="2">Locker 2</option>
-                      <option value="3">Locker 3</option>
-                      <option value="4">Locker 4</option>
-                      <option value="5">Locker 5</option>
-                      <option value="6">Locker 6</option>
-                      <option value="7">Locker 7</option>
+                      <option value="101">Locker 101</option>
```

Comment:
You could use the id as a value instead of the number. If I remember correctly, the id is used for the request and not the locker number